webview/src/plots/components/comparisonTable/ComparisonTableRows.tsx

```diff
@@ -54,9 +54,7 @@ export const ComparisionTableRows: React.FC<ComparisonTableRowsProps> = ({
           path={path}
           plots={columns.map(column => ({
             id: column.id,
-            imgs: revs[column.id]?.imgs || [
-              { errors: undefined, loading: false, url: undefined }
-            ]
+            imgs: revs[column.id]?.imgs
```

Comment:
Reviewed the way we created images on the backend and did some testing on the frontend. I don't think it's possible for `imgs` to be missing on the backend.